src/TemplateLibrary/index.js

```diff
@@ -148,7 +148,7 @@ const TemplateLibraryComponent = (props) => {
               libraryProps={libraryProps}
             />
           ))}
-        </TemplateCards>
+        </TemplateCards> : <div style={{ textAlign: 'center' }}>No results found !</div>}
```

Comment:
I think this would make more sense semantically as a `p` element instead of a `div`. I also agree with @Michael-Grover that the copy should be "No results found" without the exclamation point.

Reply:
got it @DianaLease , sending a PR in a while